src/main/java/sysc4806/project24/mini_survey_monkey/models/Question.java

```diff
@@ -0,0 +1,46 @@
+package sysc4806.project24.mini_survey_monkey.models;
+
+import jakarta.persistence.*;
+
+@Entity
+@Inheritance(strategy=InheritanceType.JOINED)
+public abstract class Question {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int questionId;
+
+    private String questionTitle;
+
+    private int questionNumber;
+
+    @ManyToOne
+    @JoinColumn(name="survey_id")
```

Comment:
Do you think it would make sense to remove the name here and let Spring auto-generate it? I think this would make the schema a little less brittle if we have to change column names in the future.

```suggestion
    @JoinColumn
```

Reply:
I don't think we end up changing them however yes it makes sense to remove it